frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -97,6 +86,159 @@ export function OverViewTab({
         return urls.insightNew({ query })
     }
 
+    return (
```

Comment:
just a refactor (moving code around) because:

> I'll need to create a separate component for the row actions since hooks can only be used at the top level of components